processor/logdedupprocessor/README.md

```diff
@@ -25,13 +25,17 @@ This processor is used to deduplicate logs by detecting identical logs over a ra
-| log_count_attribute     | string | `log_count`    | The name of the count attribute of deduplicated logs that will be added to the emitted aggregated log. |
-| timezone     | string | `UTC`    | The timezone of the `first_observed_timestamp` and `last_observed_timestamp` timestamps on the emitted aggregated log. The available locations depend on the local IANA Time Zone database. [This page](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) contains many examples, such as `America/New_York`.  |
-| exclude_fields     | []string | `[]`    | Fields to exclude from duplication matching. Fields can be excluded from the log `body` or `attributes`. These fields will not be present in the emitted aggregated log. Nested fields must be `.` delimited. If a field contains a `.` it can be escaped by using a `\` see [example config](#example-config-with-excluded-fields).<br><br>**Note**: The entire `body` cannot be excluded. If the body is a map then fields within it can be excluded. |
+| Field               | Type     | Default     | Description                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| ---                 | ---      | ---         | ---                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| interval            | duration | `10s`       | The interval at which logs are aggregated. The counter will reset after each interval.                                                                                                                                                                                                                                                                                                                                                                  |
+| condition           | string   | `true`      | An [OTTL] expression used to evaluate which log records are deduped. All paths in the [log context] are available to reference. All [converters] are available to use.                                                                                                                                                                                                                                                                                  |
```

Comment:
Is other places where we use OTTL, it's normally represented as a series of statements (eg []string) that are logically AND'd together. This helps break up more complex logic instead of including AND's in the statement itself.

The config would then look like:
```yaml
logdedup:
  condition:
    - attributes["ID"] == 1
    - resource.attributes["service.name"] == "my-service"
    - etc
```

If we do this, we should update `condition` to be plural, eg `conditions`.

Reply:
Yeah, I think OR's make sense. Then the onus in on the developer to build their conditions, and if they need multiple conditions to be met, they would construct a single statement with ANDs.

Reply:
Thanks for the response, I just pushed up a change implementing the multiple conditions. 👍